vfs/vfs.go

```diff
@@ -200,7 +201,7 @@ func New(f fs.Fs, opt *vfscommon.Options) *VFS {
 	defer activeMu.Unlock()
 	configName := fs.ConfigString(f)
 	for _, activeVFS := range active[configName] {
-		if vfs.Opt == activeVFS.Opt {
+		if reflect.DeepEqual(vfs.Opt, activeVFS.Opt) {
```

Comment:
I had to do this because it couldn't do a shallow compare of the struct with a string array as a variable (we've added `VfsUploadExclude` as a new string array to `Options`)